modules/nf-core/software/strelka/germline.nf

```diff
@@ -1,31 +1,30 @@
 // Import generic module functions
-include { initOptions; saveFiles; getSoftwareName } from './functions'
+include { initOptions; saveFiles; getSoftwareName } from './../functions'
 
-process STRELKA {
+process STRELKA_GERMLINE {
     tag "$meta.id"
     
-    label 'cpus_max'
-    label 'memory_max'
+    label 'CPUS_MAX'
+    label 'MEMORY_MAX'
```

Comment:
should we add here the 'normal' nf-core labels instead as we discussed at some point?

Reply:
yes, very good point.
Do you remember where we talked about that?

Reply:
I think some old PR, where we said it would be good to have the same labels everywhere